charts/karavi-observability/templates/cert-manager.yaml

```diff
@@ -91,45 +93,41 @@ spec:
   - otel-collector
   - otel-collector.karavi.svc.kubernetes.local
   issuerRef:
-  {{- if and (.Values.otelCollector.certificateFile) (.Values.otelCollector.privateKeyFile) }}
-    name: otel-collector-issuer
-  {{- else }}
     name: selfsigned-issuer
-  {{- end }}
     kind: Issuer
     group: cert-manager.io
 
 ---
+{{- end }}
 
+# only create Certificate for self-signed
+{{- if and (not .Values.karaviTopology.certificateFile) (not .Values.karaviTopology.privateKeyFile) }} 
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: karavi-topology
   namespace: {{ .Release.Namespace }}
 spec:
-  secretName: karavi-topology-tls
   duration: 2160h # 90d
   renewBefore: 360h # 15d
   subject:
     organizations:
     - dellemc
-  isCA: false
+  isCA: true
   privateKey:
     algorithm: RSA
     encoding: PKCS1
     size: 2048
   usages:
     - server auth
     - client auth
+    - signing
   dnsNames:
   - karavi-topology
   - karavi-topology.karavi.svc.kubernetes.local
```

Comment:
Same here, the FQDN should be `karavi-topology.{{ .Release.Namespace }}.svc.kubernetes.local`